test/testutils/wrappers.go

```diff
@@ -288,3 +293,24 @@ func MakeLeaderPodSpecWithTPUResource() corev1.PodSpec {
 		Subdomain: "default",
 	}
 }
+
+func RawLWSTemplate(lws *leaderworkerset.LeaderWorkerSet) runtime.RawExtension {
```

Comment:
I am not sure we should be using a function like this one in our tests; our tests shouldn't rely on a copy the function that we partially try to test

Reply:
In this case it is not needed, but there are cases where we do need to duplicate functions as otherwise a dependency cycle is created.

Reply:
remove the commented out code please.